openmetadata-ui/src/main/resources/ui/src/pages/tags/TagsPage.style.less

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+.tags-page-left-panel {
+  height: calc(100vh - 90px);
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
```

Comment:
Instead specific class for left panel can you make some generic style for left panel?

Reply:
made generic class left-panel-container for styling left panel